docs/products/kafka/kafka-mirrormaker/howto/integrate-external-kafka-cluster.rst

```diff
@@ -20,4 +20,108 @@ You can define an external Apache Kafka® service integration endpoint using the
+
+The set of topics created by Apache Kafka® MirrorMaker2 to source cluster is the following.
+
+.. list-table::
+  :header-rows: 1
+
+  * - Topic
+    - | Replication factor
+      | (default value)
+    - | Partitions
+      | (default value)
+    - | Cleanup policy
+      | (required policy)
+  * - ``__consumer_offsets``
+    - 3
+    - 50
+    - compact
+  * - ``heartbeats``
+    - 3
+    - 1
+    - compact
+  * - ``mm2-configs.<target cluster alias>.internal``
+    - 3
+    - 1
+    - compact
+  * - | ``mm2-offset-syncs.<target cluster alias>.internal``
+      | see `Offsets sync topic location`_
+    - 3
+    - 1
+    - compact
+  * - ``mm2-offsets.<target cluster alias>.internal``
+    - 3
+    - 25
+    - compact
+  * - ``mm2-status.<target cluster alias>.internal``
+    - 3
+    - 5
+    - compact
+
+
+The set of topics created by Apache Kafka® MirrorMaker2 to target cluster is the following:
+
+.. list-table::
+  :header-rows: 1
+
+  * - Topic
+    - | Replication factor
+      | (default value)
+    - | Partitions
+      | (default value)
+    - | Cleanup policy
+      | (required policy)
+  * - ``__consumer_offsets``
+    - 3
+    - 50
+    - compact
+  * - ``heartbeats``
+    - 3
+    - 1
+    - compact
+  * - ``<source cluster alias>.checkpoints.internal``
+    - 3
+    - 1
+    - compact
+  * - ``<source cluster alias>.heartbeats``
+    - 3
+    - 1
+    - compact
+  * - ``mm2-configs.<source cluster alias>.internal``
+    - 3
+    - 1
+    - compact
+  * - ``mm2-offsets.<source cluster alias>.internal``
+    - 3
+    - 25
+    - compact
+  * - ``mm2-status.<source cluster alias>.internal``
+    - 3
+    - 5
+    - compact
+
+
+If using heartbeat emitting to source cluster, configuration is called ``emit_backward_heartbeats_enabled``, the read and write is required to ``mm2-offsets.<target cluster alias>.internal`` topic at the external source cluster. When disabled the topic and permissions are not required at the source cluster.
+Similarly if using heartbeat emitting to target cluster, configuration is called ``emit_heartbeats_enabled``, the read and write is required to ``mm2-offsets.<source cluster alias>.internal``. When disabled the topic and permissions are not required at the target cluster.
+
+
+.. note::
+   Summarizing the requirements for external Kafka cluster integration, configure the ACLs for both the source and target cluster such that the Apache Kafka® MirrorMaker 2 service can describe and create topics, as well as produce and consume messages.
+
+
+Offsets sync topic location
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Offsets sync topic can be either in source cluster or target cluster based on the configuration value of ``offset-syncs.topic.location``. Default location is ``source``. Connecting user must have permission to read and write this topic.
+
+If location is configured as ``target`` this topic will be created to the target cluster.
```

Comment:
```suggestion
If the location is configured as ``target, `` this topic will be created in the target cluster.
```

Reply:
Added some minor suggestions to the content (mostly trying to use active voice as much as possible).

Reply:
Done.